service/src/main/java/eu/europa/ec/fisheries/ers/service/bean/FishingTripServiceBean.java

```diff
@@ -634,6 +640,31 @@ FishingTripResponse buildFishingTripSearchRespose(Set<FishingTripId> fishingTrip
         return response;
     }
 
+    private CronologyDTO buildCronologyDTO(String tripId) throws ServiceException {
+        CronologyDTO cronologyDTO = new CronologyDTO();
+        FishingActivityQuery query = new FishingActivityQuery();
+        Map<SearchFilter, String> searchCriteriaMap = new EnumMap<>(SearchFilter.class);
+        searchCriteriaMap.put(SearchFilter.TRIP_ID, tripId);
+        query.setSearchCriteriaMap(searchCriteriaMap);

```

Comment:
The last 3 lines can be replaced by:

    query.setSearchCriteriaMap(Collections.singletonMap(SearchFilter.TRIP_ID, tripId));

Just saying...